### DISCUSSION TITLE
Support for finer playback control

### DISCUSSION THREAD

**Discussion by TheByteSmasher** [yermak/AudioBookConverter#579] [Q&A]
Hi! any chance to make the audio controls finer? or even just play the selected chapter? Thanks

**Comment by yermak** [comment]
You can play selected chapter, by selecting specific file in the chapter.
I know that playback feature is quite ugly, but I expected that it only will be used to hear the begining of the file to understand chapter number, etc.
If you would like other controls please describe them as separate feture request.
P.S. Moving this to discussions
